components/expenses/ExpenseAdminActions.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Mutation } from '@apollo/react-components';
```

Comment:
🍨 I tend to find hooks more elegant because they're pure JS.

Reply:
I prefer them too in most cases, but the components are quite useful when you have conditional rendering, ie:

```jsx
<div>
  ...
  {hasData &&  (
    <Query query={...}
       ...
    </Query>
  })
<div>
```

Reply:
True. I was going to say that this implies the necessity of a subcomponent, but it does it explicitly. 😂